src/BGIter.js

```diff
@@ -0,0 +1,43 @@
+class BGIter {
+  constructor(interval = 1000, iterations = 10) {
+    this.interval = interval;
+    this.iterations = iterations;
+    this.usedColors = {};
+    this.count = 0;
+    this.timeout;
+  }
+
+  static setBg(color) {
+    document.body.style.backgroundColor = color;
+  }
+
+  static genColor() {
```

Comment:
Using a static class method here clearly works just fine, but could you tell me why you preferred this approach over having a normal class method?

Reply:
The function doesn't use any class members, it could make sense to call either this or the `setBg` function without an instance of the class. I declared them static so that would be possible.